### PR TITLE
Add Bundle ID to event tracking

### PR DESCRIPTION
**Pull request by BenjiLee**
Is it as simple as this?

### CODE REVIEW

File: ecommerce/extensions/basket/views.py [hunk 1]
```diff
@@ -336,9 +336,11 @@ def _get_payment_processors_data(self, payment_processors):
 
     def get(self, request, *args, **kwargs):
         basket = request.basket
+        bundle_id = request.GET.get('bundle', None)
```
Comment (BenjiLee) [inline ecommerce/extensions/basket/views.py]: Note to self: Only the UTM params are passed as query parameters to BasketSummary from BasketAdd. Perhaps I can store the bundle ID in the request cache?

Comment (BenjiLee) [inline ecommerce/extensions/basket/views.py]: Note to self: Only the UTM params are passed as query parameters to BasketSummary from BasketAdd. Perhaps I can store the bundle ID in the request cache?

Comment (robrap) [reply ecommerce/extensions/basket/views.py]: Isn't the bundle id stored in a basket attribute?

Comment (BenjiLee) [reply ecommerce/extensions/basket/views.py]: yes. yes it is.
```
SELECT value_text from ecommerce.basket_basketattribute WHERE attribute_type_id=2;
```
returns the bundle-id

**Comment by robrap** [comment]
@BenjiLee: Giving up? :(

**Comment by robrap** [comment]
@BenjiLee: Can you remind me what you couldn't do and what you were trying to fix with this?  I spoke to the Data Science team, and they think they have ways, so just curious.  Thank you!

**Comment by BenjiLee** [comment]
7 years later... 

I think the original goal was to see how our 20s to 2s performance improvement for programs impacted the bounce rate of programs. After we made the improvement, it was found that we do not track programs separately from courses so we couldn't answer that question. This PR was to add the bundle-id so that we can see just programs in an analytics filter/query.

**Comment by robrap** [comment]
Thank you @BenjiLee. I see that this was for ‘Cart Viewed’ and ‘Checkout Step Viewed’. Lily said they have generally been determining this based on the product type. Is that what you were told, but the joins were too complicated, or were you trying to figure it out another way? In any case, I was seeing if we could have the Data Science team support getting this through if it would be useful to them.

**Comment by BenjiLee** [comment]
I was told that it was possible to get the information I was looking for, I just needed to get access to the data warehouse pieces and learn how to do the query. I thought this was a a high barrier to entry for finding out something like this, thus this PR.

**Comment by robrap** [comment]
Thanks @BenjiLee.
Note: @cpennington may try to actually land some of your old PRs, including the apply CustomApplicator everywhere, which still hasn't landed.  If you have any other PRs you closed that @cpennington should know about, please feel free to share. :)

**Comment by cpennington** [comment]
I just chatted with @lxedx, and I don't think we need to add this to segment. We can analyze the downstream data and know in the data warehouse whether a user was purchasing a bundle or not.

@BenjiLee (or @robrap), when did the performance change land? I'd be interested in actually doing the before/after analysis, because the question of whether improving perf improves revenue is important to us.

**Comment by robrap** [comment]
@cpennington: I forwarded you an email thread titled "Learner Performance Team Update for Ecommerce".

**Pull request closed without merging**